modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpics.scala

```diff
@@ -209,16 +209,32 @@ object TcsControllerEpics extends TcsController {
         setOiwfs, EpicsTcsConfig.oiwfs ^|-> GuiderConfig.detector)
     ).collect{ case Some(x) => x }
 
+
+  def tagIso[B, T]: Iso[B@@T, B] = Iso.apply[B@@T, B](x => x)(tag[T](_))
```

Comment:
Nice!

Reply:
To be used cautiously, with the risk of erasing the benefits of `tag`